data/json/monsters/reptile_amphibian.json

```diff
@@ -195,4 +195,49 @@
     "path_settings": { "max_dist": 5 },
     "flags": [ "SEES", "SMELLS", "WARM", "VENOM", "SWIMS" ]
   }
+{
+    "id": "mon_turtle_giant",
+    "type": "MONSTER",
+    "name": { "str": "giant turtle" },
+    "description": "A horrifically large and ulcerous snapping turtle. In the air above its primordial shell it interweaves its unnaturally elongated and flexible neck and tail with unnervingly elegant complexity.",
```

Comment:
thank you for all these suggestions. this feedback is exactly what i was looking for.

i chose "ulcerous" because ulcerative shell disease ("shell rot") is fairly common. the use of "primordial" here is not strictly correct, but it's meant to capture the ancient-looking quality of the turtle's anatomy. ill try to think of a better word.